admin/src/components/Tables/SearchUserTable.vue

```diff
@@ -79,6 +79,9 @@
             <b-tab :title="$t('transactionlink.name')" :disabled="row.item.deletedAt !== null">
               <transaction-link-list v-if="!row.item.deletedAt" :userId="row.item.userId" />
             </b-tab>
+            <b-tab :title="$t('userRole.tabTitle')">
+              <change-user-role-formular :item="row.item" @updateDeletedAt="updateDeletedAt" />
```

Comment:
```suggestion
              <change-user-role-formular :item="row.item" @updateIsAdmin="updateIsAdmin" />
```

Reply:
Where is that method?

Reply:
Oh!
Very good catch !!! Thanks a lot !!! 🙏🏼

I added it 👍🏼